File: python3/2425.py
class Solution:
    def xorAllNums(self, n1: List[int], n2: List[int]) -> int:
        res = 0
        if len(n1) % 2 != 0:
            for i in n2:
                res ^= i
        if len(n2) % 2 != 0:
            for i in n1:
                res ^= i
        return res

File: java/2425.java
class Solution {
    public int xorAllNums(int[] n1, int[] n2) {
        int res = 0;
        if (n1.length % 2 != 0)
            for (int i : n2)
                res ^= i;
        if (n2.length % 2 != 0)
            for (int i : n1)
                res ^= i;
        return res;
    }
}
